FrontMen/fm-hours
# feat(records): grey ou part time days

# Changes

## Related issues

Resolves #12 

## Added

- Add `isPartTime` to `WeekDay` type
- Part-time workdays have a slightly darker background

## Removed

N/A

## Changed

N/A

## How to test

- If you don't have a part-time work-scheme you can either ask to temporarily change it on Bridge to one of the admins or mockup the response from the work-schema-service
- Check if the background for part-time days have a different color

## Screenshots

<img width="1904" alt="Screenshot 2021-07-23 at 09 03 34" src="https://user-images.githubusercontent.com/45885054/126749567-b9d6c52d-1090-48b0-b678-a7a167f02344.png">
